### PR TITLE
changed the URL that is passed to Heroku

### PR DESCRIPTION
**Pull request by jroberts235**
[PT-#1234](https://www.pivotaltracker.com/story/show/1234)

## Type of Changes

- [ ] Bug fix _(a non-breaking change which fixes an issue)_
- [ ] New feature _(a non-breaking change which adds functionality)_
- [ ] Breaking change _(fix or feature that changes existing functionality)_
- [ ] Bump dependencies
- [ ] Automated Testing / Missing tests
- [ ] Documentation

## Checklist:

- [ ] Yes, I have added tests to cover my changes.
- [ ] No, We don't have a test suite in this project yet.
- [ ] My change requires a change to the documentation.
- [ ] My change requires release strategy _(uncomment the Release strategy below)_

cc/

### CODE REVIEW

File: heroku-app-create/heroku-app-create.py [hunk 1]
```diff
@@ -268,9 +271,9 @@ def heroku_paginated_get_json_array( url, **kwargs ):
 
 # GitHub Related Functions #####################################################
 
-def get_download_url( repo, branch, token ):
+def get_download_url( repo, branch, GHA_USER_TOKEN, GITHUB_USER ):
     # pulls the 302 location out of the redirect
-    return API_URL_GITHUB+'/repos/'+repo+'/tarball/'+urllib.parse.quote(branch)+'?access_token='+token
+    return f'https://{GITHUB_USER}:{GHA_USER_TOKEN}@api.github.com/repos/{GITHUB_USER}/{repo}/tarball/{urllib.parse.quote(branch)}'
```
Comment (josh-schlanger-therealreal) [inline heroku-app-create/heroku-app-create.py:276]: is there a reason you are using the actual URL - "api.github.com" vs using the variable? @jroberts235 

Also, not an expert, but is the "f" at the beginning of the line necessary.  Just making sure its not a typo, but think its needed.

Comment (jroberts235) [reply heroku-app-create/heroku-app-create.py:276]: Because the global variable is used elsewhere in the code for other things.

Comment (josh-schlanger-therealreal) [reply heroku-app-create/heroku-app-create.py:276]: OK

**Review by josh-schlanger-therealreal — APPROVED** [review]

File: heroku-app-create/heroku-app-create.py (hunk 1 above)
Comment (chrisghill) [reply heroku-app-create/heroku-app-create.py:276]: I don't think we want the {GITHUB_USER} in the middle of the URL. That should be `therealreal` not `trr-devops`.

**Pull request merged**